force_bdss/tests/probe_classes/i_evaluator.py

```diff
@@ -0,0 +1,21 @@
+from traits.api import (
+    provides,
+    Instance
+)
+
+from force_bdss.mco.i_evaluator import IEvaluator
+
+from force_bdss.mco.base_mco_model import BaseMCOModel
+
+
+class DummyMCOModel(BaseMCOModel):
+    pass
+
+
+@provides(IEvaluator)
+class ProbeIEvaluator:
```

Comment:
Nitpick: this is not an `Interface` class, so the naming convention would be to drop the `I` here and call it `ProbeEvaluator`.

Similarly, this file should probably be called `evaluator.py` rather than `i_evaluator.py`